app/models/katello/glue/pulp/repo.rb

```diff
@@ -356,6 +356,192 @@ def purge_empty_groups_errata
         end
       end
 
+      def index_db_errata(force = false)
```

Comment:
Should these changes be included in this PR?  (e.g. addition of index_db_[errata|rpms|files|puppet_modules|package_groups]...etc)